pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md

```diff
@@ -28,23 +28,23 @@ To successfully adapt your applications, you must have:
 Check for the following on your existing `cluster.yaml`:
 
 - One or more of `spec.kubernetes.networking.noProxy`, `spec.kubernetes.networking.httpProxy` or `spec.kubernetes.networking.httpsProxy` is set in `ClusterConfiguration`.
-- Enabled `gatekeeper` with custom `values` field in `ClusterConfiguration` in `cluster.yaml`.
+    - Enabled `gatekeeper` with custom `values` field in `ClusterConfiguration` in `cluster.yaml`.
 
 If none of the conditions apply to your cluster, then you can skip to next section.
 
-1.  Because Kommander 2.0+ uses flux to manage applications, we need to configure the gatekeeper `mutatingwebhookconfigurations` (which is a cluster-scoped resource) such that it allows `dry-run` calls, which are required by the flux kustomize controller to calculate the diff of a resource. In order to do this:
+1.  Because Kommander 2.0+ uses Flux to manage applications, we need to configure the Gatekeeper `mutatingwebhookconfigurations` (which is a cluster-scoped resource) such that it allows `dry-run` calls, which are required by the Flux kustomize controller to calculate the difference of a resource. In order to do this:
```

Comment:
```suggestion
1.  Because Kommander 2.0+ uses Flux to manage applications, you must configure the Gatekeeper `mutatingwebhookconfigurations` (which is a cluster-scoped resource) to allow `dry-run` calls. They are required by the Flux kustomize controller to calculate the difference of a resource. To do this:
```

Reply:
updated